addons/portal/models/portal_mixin.py

```diff
@@ -34,20 +43,37 @@ def _get_access_token(self):
         field = self._get_access_token_field()
         return self[field] if field else False
 
+    @api.model_cr_context
+    def _init_column(self, column_name):
```

Comment:
Add a compute method (with store = True) on access_token can do the job ? or it will compute for every existing record, maybe ? Or we call a get_access_token when sharing ?